dashboard/config/foorm/library/surveys/pd/ayw_module_1.0.json

```diff
@@ -136,6 +136,7 @@
       {
        "type": "comment",
        "name": "other_activities_m1",
+              "maxLength": 4000,
```

Comment:
you can leave this in here but I added a fix that will applies a blanket maximum length for all free responses